<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>F44026123</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.png</normaloff>icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>70</y>
      <width>40</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>70</y>
      <width>211</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="card_3">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>150</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>03</string>
    </property>
   </widget>
   <widget class="QPushButton" name="card_1">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>150</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>01</string>
    </property>
   </widget>
   <widget class="QPushButton" name="card_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>150</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>02</string>
    </property>
   </widget>
   <widget class="QPushButton" name="card_4">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>150</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>04</string>
    </property>
   </widget>
   <widget class="QPushButton" name="card_5">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>190</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>05</string>
    </property>
   </widget>
   <widget class="QPushButton" name="card_6">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>190</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>06</string>
    </property>
   </widget>
   <widget class="QPushButton" name="card_7">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>190</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>07</string>
    </property>
   </widget>
   <widget class="QPushButton" name="card_8">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>190</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>08</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="card_display_1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>250</y>
      <width>121</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="card_display_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>250</y>
      <width>121</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="card_display_3">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>250</y>
      <width>121</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="card_display_4">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>250</y>
      <width>121</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="playButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>480</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>PLAY!</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>440</y>
      <width>98</width>
      <height>15</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label2">
     <property name="text">
      <string>Player : </string>
     </property>
    </widget>
    <widget class="QLabel" name="player">
     <property name="text">
      <string>Player : </string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
